partials/payment-term.html

```diff
@@ -9,9 +9,14 @@
 	</label>
 	{{/each}}
 	<div class="ncf__payment-term__legal">
-		With all subscription types, we will automatically renew your subscription using the payment method provided unless you
-		cancel before your renewal date. We will notify you at least 14 days in advance of any changes to the price in your subscription that would apply upon next
-		renewal. Find out more about our cancellation policy in our
-		<a href="https://help.ft.com/help/legal-privacy/terms-conditions/" title="FT Legal Terms and Conditions help page">Terms &amp; Conditions</a>.
+		<p>
+			With all subscription types, we will automatically renew your subscription using the payment
+			method provided unless you cancel before your renewal date.
+		</p>
+		<p>
+			We will notify you at least 14 days in advance of any changes to the price in your subscription
+			that would apply upon next renewal. Find out more about our cancellation policy in our
+			<a href="https://help.ft.com/help/legal-privacy/terms-conditions/" title="FT Legal Terms and Conditions help page">Terms &amp; Conditions</a>.
```

Comment:
Two for one https://trello.com/c/EUOm5Yeu/1120-1-missing-link-icon :D

```suggestion
			<a class="ncf__link--external" href="http://help.ft.com/help/legal-privacy/terms-conditions/" title="FT Legal Terms and Conditions help page" target="_blank" rel="noopener">Terms &amp; Conditions</a>
```